packages/common-sdk/src/web3/ata-util.ts

```diff
@@ -22,20 +22,23 @@ import { EMPTY_INSTRUCTION } from "./transactions/types";
  * @param ownerAddress The user's public key
  * @param tokenMint Token mint address
  * @param wrappedSolAmountIn Optional. Only use for input/source token that could be SOL
+ * @param payer Payer that would pay the rent for the creation of the ATAs
  * @returns
  */
 export async function resolveOrCreateATA(
   connection: Connection,
   ownerAddress: PublicKey,
   tokenMint: PublicKey,
   getAccountRentExempt: () => Promise<number>,
-  wrappedSolAmountIn = new u64(0)
+  wrappedSolAmountIn = new u64(0),
+  payer?: PublicKey
```

Comment:
if we know the payer will default to owner, we could make it non-optional, and just do `payer: PublicKey = ownerAddress` in the params?

Reply:
yeah that's a good suggestion. making the change